DBugVision/dbug_image.py

```diff
@@ -1,10 +1,33 @@
 from dbug_contour import DbugContour
+import cv2
+import numpy as np
 
 
 class DBugAbstractImage(object):
-    #  TODO: add docs
-    def __init__(self, image):
-        pass
+    """
+    The abstract class of an image (e.g: frame taken from a camera), all types of images should inherit from

```

Comment:
This is REALLY nice! i didn't know it existed. But here i don't think it is needed (after i used the self.____class____) this is not really an abstract class rather a mother class. I changed the docs and code to match.